core/web/evm_forwarders_controller.go

```diff
@@ -56,8 +59,10 @@ func (cc *EVMForwardersController) Track(c *gin.Context) {
 		jsonAPIError(c, http.StatusUnprocessableEntity, err)
 		return
 	}
-	orm := forwarders.NewORM(cc.App.GetSqlxDB(), cc.App.GetLogger(), cc.App.GetConfig().Database())
-	fwd, err := orm.CreateForwarder(request.Address, *request.EVMChainID)
+	orm := forwarders.NewORM(cc.App.GetSqlxDB())
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	fwd, err := orm.CreateForwarder(ctx, request.Address, *request.EVMChainID)
```

Comment:
Same as Jordan's suggestion above, you can pass `gin`'s context on all these requests